src/ResourceManagement/RedisCache/Microsoft.Azure.Management.Redis/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.1.1-preview",
+  "version": "3.0.0-preview",
```

Comment:
API version did change from "2015-08-01" to "2016-04-01". Also there is parameter removal for vnet